src/content/developers/docs/spin-up-your-node/index.md

```diff
@@ -0,0 +1,146 @@
+For testing purposes, user might prefer running client on one of testnet networks. Overview of supported networks and sync modes can be found in [Clients section](/en/developers/docs/nodes-and-clients/#clients). 
+Most important factor which depends on these settings is how much disk space will be required for database, therefore what drive will be needed.
+
+### Environment and hardware {#environment-and-hardware}
+
+#### Local or cloud {#local-vs-cloud}
+Ethereum clients are able to run on consumer grade computers and don't require special hardware like mining for example. Therefore user has various options where to deploy the instance depending on individual needs. 
+To simplify the case, let's divide our possibilities to running node on local physical machine and renting server in the cloud: 
+- Cloud
+    - Providers offer high server uptime, static public IP addresses
+    - Getting dedicated or virtual server can be more comfortable then building own
+    - Trade off is trusting a third party - server provider
+    - Because of required storage size for full node, price of a rented server might get high
+- Own hardware
+    - More trustless and sovereign approach
+    - One time investment 
+    - An option to buy preconfigured machines 
+    - Requires to physically prepare the machine, maintain it and solve possible issues with networking
+
+Both options have different advantages summed up above. If you are looking for a cloud solution, in addition to many traditional cloud computing providers, there are also services focused on deploying nodes, for example [QuikNode](https://www.quiknode.io/), [Blockdaemon](https://blockdaemon.com), [LunaNode](https://www.lunanode.com/). 
+Censorship resistant and decentralized network should not rely on cloud providers infrastructure. For running Ethereum node locally, check Hardware section below. Easiest options are preconfigured machines like [DappNode](https://dappnode.io/) or [Avado](https://ava.do/). 
+
+#### Hardware {#hardware}
+
+Minimum and recommended requirements are summed up in [Nodes and clients page section](/en/developers/docs/nodes-and-clients/#requirements). 
+Generally, modest computing power should be enough, bottleneck is usually speed of a drive. During initial sync, Ethereum client performs a lot of read/write operations, therefore SSD is strongly recommended. Client might even [not be able to sync current state on HDD](https://github.com/ethereum/go-ethereum/issues/16796#issuecomment-391649278) and get stuck few blocks behind mainnet. 
+Some clients offer builds for different architectures which makes possible to run it on [SCBs with ARM](/en/developers/docs/nodes-and-clients/#ethereum-on-a-single-board-computer/). Another easy way is using [Ethbian](https://ethbian.org/index.html) operating system for Raspberry Pi 4 which enables [run Ethereum client by simply flashing SD card](/en/developers/tutorials/run-node-raspberry-pi/).
```

Comment:
"You can run some clients on a single board computer using ARM"

Can we state which clients here?
Can we avoid acronyms wherever possible (unless the acronym is more popular).